lib/eslintPlaywrightConfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "plugins": ["eslint-plugin-playwright"],
-  "extends": ["plugin:playwright/recommended", "./eslintNodeConfig.json", "eslint-config-prettier"],
+  "extends": ["plugin:playwright/recommended"],
```

Comment:
Node.js can no longer be assumed. Could be Bun or Node.

Reply:
`eslint-config-prettier` only belongs as the last `extends` entry, so it should only ever be in one of the environment configs:
- Node
- Bun
- Browser